### DISCUSSION TITLE
Update-check interval

### DISCUSSION THREAD

**Discussion by soa-jrt** [centerclick/feedback#117] [Q&A] [answered]
How often does an NTP270 running 2.18.3 phone home to check for updates? I see that 2.19.0 is available, and dated 20241107. Our appliance shows the following in the JSON output:
-   "SWVER": "2.18.3"
-   "SWVER_LATEST": "check-pending"
-   "UPGRADE_CHECK": "install"
-   "uptime": "11 days"

I could force an update-check, but I've been waiting to see how long it will take to check/notice/install on its own. 
How do I determine if it has been checking and failing?

**Comment by dave4445** [comment, marked as answer]
Once a day between 1AM-5AM in the configured timezone.
However, during the first 7 days after release, server-side, we roll out updates to devices with auto-upgrade enabled slowly, only a subset of devices get the update each day.

**Comment by dave4445** [comment]
Also note that if you see `check-pending` for more than a day, then the check is failing.  You can run an `upgrade` command to see if there is a connectivity problem.

**Reply by soa-jrt** [reply]
ty. I like the idea of dribbling out the update.
Running the 'upgrade' command reveals the problem, and explains why we are "check-pending" for 11 days . . we have a MITM outbound proxy with a internally-signed certificate :(
Is there a way to update the list of system-wide trusted certificates?

**Reply by dave4445** [reply]
That would seem like a slippery slope, if you want to mirror the file locally you can `upgrade <url>` for now

**Reply by soa-jrt** [reply]
Ok. I see how that syntax should work. I'll give it a try.
